README.md

```diff
@@ -30,18 +30,31 @@ There is currently one public instance running at
 
 # Development / Running
 
+## Dependencies
+
+If you've yet to install `bower` and `gulp`, run
+
+``` $ npm install -g bower ```
+
+``` $ npm install -g gulp ```
+
+
+## Install and Run
+
 To install and build static assets:
 
 ``` $ npm install ```
 
+``` $ mkdir build ```
+
+``` $ cd static/ && bower install && cd .. ```
```

Comment:
@jimmcgaw Without this I wasn't getting jQuery and Bootstrap, and running `gulp inject` directly was giving me problems.  How's this for a solution?